src/EFCore.PG.FuzzyStringMatch/Extensions/NpgsqlFuzzyStringMatchDbFunctionsExtensions.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+
+namespace Microsoft.EntityFrameworkCore
+{
+    public static class NpgsqlFuzzyStringMatchDbFunctionsExtensions
+    {
+        /// <summary>
+        /// The soundex function converts a string to its Soundex code.
+        /// </summary>
+        /// <remarks>
+        /// The method call is translated to <c>soundex(text)</c>.
+        ///
+        /// See https://www.postgresql.org/docs/current/fuzzystrmatch.html.
+        /// </remarks>
+        public static string FuzzyStringMatchSoundex(this DbFunctions _, string text) =>
+            throw new NotSupportedException();
+
+        /// <summary>
+        /// The difference function converts two strings to their Soundex codes and
+        /// then returns the number of matching code positions. Since Soundex codes
+        /// have four characters, the result ranges from zero to four, with zero being
+        /// no match and four being an exact match.
+        /// </summary>
+        /// <remarks>
+        /// The method call is translated to <c>difference(source, target)</c>.
+        ///
+        /// See https://www.postgresql.org/docs/current/fuzzystrmatch.html.
+        /// </remarks>
+        public static int FuzzyStringMatchDifference(this DbFunctions _, string source, string target) =>
+            throw new NotSupportedException();
+
+        /// <summary>
+        /// Returns the Levenshtein distance between two strings.
+        /// </summary>
+        /// <remarks>
+        /// The method call is translated to <c>levenshtein(source, target)</c>.
+        ///
+        /// See https://www.postgresql.org/docs/current/fuzzystrmatch.html.
+        /// </remarks>
+        public static int FuzzyStringMatchLevenshtein(this DbFunctions _, string source, string target) =>
+            throw new NotSupportedException();
+
+        /// <summary>
+        /// Returns the Levenshtein distance between two strings.
+        /// </summary>
+        /// <remarks>
+        /// The method call is translated to <c>levenshtein(source, target, ins_cost, del_cost, sub_cost)</c>.
+        ///
+        /// See https://www.postgresql.org/docs/current/fuzzystrmatch.html.
+        /// </remarks>
+        public static int FuzzyStringMatchLevenshtein(this DbFunctions _, string source, string target, int ins_cost, int del_cost, int sub_cost) =>
```

Comment:
Convert to C# naming, let's also use full names (i.e. insertionCost, deletionCost, substitutionCost)

Reply:
Fixed with fdb747646b99958c3e0d84414c2ed6ec26021eb8.